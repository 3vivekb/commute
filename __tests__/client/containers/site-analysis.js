/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'
import {Provider} from 'react-redux'

import {makeMockStore, mockStores} from '../../test-utils/mock-data.js'

import SiteAnalysis from '../../../client/containers/site-analysis'

const mockStore = makeMockStore(mockStores.init)

describe('Container > SiteAnalysis', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <SiteAnalysis />
      </Provider>
    )
    expect(mountToJson(tree.find(SiteAnalysis))).toMatchSnapshot()
  })
})
