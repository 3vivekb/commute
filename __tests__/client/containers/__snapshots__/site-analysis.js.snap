exports[`Container > SiteAnalysis renders correctly 1`] = `
<Connect(SiteAnalysis)>
  <SiteAnalysis
    dispatch={[Function]}
    user={Object {}}>
    <h1>
      Site Analysis
    </h1>
  </SiteAnalysis>
</Connect(SiteAnalysis)>
`;
